### ISSUE TITLE
Can't filter on default accuracy on website

### ISSUE THREAD

**Issue by amueller** [openml/OpenML#885]
The default accuracy filter is broken, I think because the quality is not computed.

**Comment by joaquinvanschoren** [comment]
@janvanrijn Database search confirms that this is not computed.

**Comment by janvanrijn** [comment]
Could be that we removed it, since it is (without CV) the same as `majorityClassPercentage`. Shall we remove `DefaultAccuracy` from the DB?

**Comment by joaquinvanschoren** [comment]
OK for me

**Comment by joaquinvanschoren** [comment]
@amueller Is this the result you were expecting?
https://www.openml.org/search?type=data&sort=qualities.MajorityClassPercentage&order=asc

**Comment by amueller** [comment]
yeah I might have even argued that this is a better way to say it at some point ;)
It would be good to replace it in the UI though. (making it a frontend issue again ;)

**Comment by joaquinvanschoren** [comment]
I've deleted DefaultAccuracy from the search index.
@janvanrijn: shall we remove it from the database as well?

**Comment by joaquinvanschoren** [comment]
Ah, it was also in the filter options. I'll replace it.

**Comment by janvanrijn** [comment]
Yes. Also from the sql file.

**Comment by amueller** [comment]
Doesn't seem to work?
